### ISSUE TITLE
Fix `media-feature-range-notation` false positives for multiple queries and `except: exact-value`

### ISSUE THREAD

**Issue by Josh-Cena** [stylelint/stylelint#8826]
### What minimal example or steps are needed to reproduce the bug?

```css
@media (width: 100px) {
  /* … */
}

@media (width >= 100px) {
  /* … */
}

@media (width: 100px) {
  /* … */
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "media-feature-range-notation": ["context", { "except": ["exact-value"] }]
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBbGUCWBDABACgHcMoAXAC3hwEYAGGgBwA8BKHYAHQDsccB6AKhyAyAhz9eXAL5cuqdNnxFSZHAD4AvNTpNWHbn0Eixk6Z1mZchYuUq0GLNlx4Dho8ZymcQAGhAAzDAA2MAByWGgIIDCMYfRBAHRgAM6J3uAQnP4A5hG6POwgAE4ArkGJ+ZS5PDj5aOYAtL4wWCRFBTB1BVicme2cECTNGOnlOADa+ZCcJFEk+V5s1ZGMYDD0swhj+VFYYCR1AG5YAUUw+QC6OBJnjpfGnj6TWQBiEAUozREAVonpqbD0KUQlXyiRIAE8ggEMFMRvkAs0YKC5jcQeDIdC9o8MJk6qCulAsAUoLCQPDpkiQJIQBIgA)

### Which Stylelint-related dependencies are you using?

Latest

### What did you expect to happen?

No problems

### What actually happened?

Errors:

```
5:8-24 error Expected "prefix" media feature range notation (media-feature-range-notation)
9:8-22 error Expected "context" media feature range notation (media-feature-range-notation)
```

### Do you have a proposal to fix the bug?

I don't know what exactly is causing this error, but I know that the first condition, `@media (width: 100px)`, which has no errors, must be included. If you remove this condition, then the next two no longer cause errors.

**Comment by jeddy3** [comment]
@Josh-Cena Thanks for the report.

Reproducible with:

```css
@media (width: 1px) {}
@media (width: 1px) {}
```

And:

```css
@media (width: 1px), (width: 2px) {}
```

I've opened a PR to fix it.

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBbGUCWBDABACgHcMoAXAC3hwEYAGGgBwA8BKHYAHQDsccB6AKhyAyAhz9eXAL5cuqdNnxFSZHAD4AvNTpNWHbn0Eixk6Z1mZchYuUq0GLNlx4Dho8ZymcQAGhAAzDAA2MAByWGgIIDCMYfRBAHRgAM6J3uAQnP4A5hG6POwgAE4ArkGJ+ZS5PDj5aOYAtL4wWCRFBTB1BVicme2cECTNGOnlOADa+ZCcJFEk+V5s1ZGMYDD0swhj+VFYYCR1AG5YAUUw+QC6OBJnjpfGnj6TWQBiEAUozREAVonpqbD0KUQlXyiRIAE8ggEMFMRvkyCQSAD4LxePQANaZOL0ApxTgwAi8UEQmBQqZIKAANhoWCwACYAEZzG4g8GQ6F7R4YTJ1UFdKBYApQWEgALNGCg-KSEASIA).
